authz/rbac_translator.go

```diff
@@ -36,6 +37,8 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
+const typedUrlPrefix = "grpc.authz.audit_logging/"
```

Comment:
Suggest adding a comment that this is used to convert the config into a typedstruct.

Reply:
Fixed, PTAL